rpcd/playbooks/repo-fetcher.yml

```diff
@@ -0,0 +1,67 @@
+---
+- name: lay the repos down
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  user: root
+  tasks:
+    - name: lay the repos down
+      git:
+        dest: "{{ item.value.dest }}"
+        repo: "{{ item.value.repo }}"
+        version: "{{ item.value.version }}"
+      with_dict: "{{repos}}"
+    - name: generate rpcd ansible.cfg
+      copy:
+        dest: /opt/rpc-openstack/rpcd/playbooks/ansible.cfg
```

Comment:
I think the point is that we supply this in the repo when it is cloned:

https://github.com/rcbops/rpc-openstack/blob/master/rpcd/playbooks/ansible.cfg

If you want to generate it dynamically, remove the supplied one